homeassistant/components/wiz_light/light.py

```diff
@@ -0,0 +1,421 @@
+"""WiZ Light integration."""
+from datetime import timedelta
+import logging
+
+from pywizlight import SCENES, PilotBuilder, wizlight
+from pywizlight.exceptions import (
+    WizLightConnectionError,
+    WizLightNotKnownBulb,
+    WizLightTimeOutError,
+)
+import voluptuous as vol
+
+# Import the device class from the component
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP,
+    ATTR_EFFECT,
+    ATTR_HS_COLOR,
+    ATTR_RGB_COLOR,
+    PLATFORM_SCHEMA,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR,
+    SUPPORT_COLOR_TEMP,
+    SUPPORT_EFFECT,
+    LightEntity,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import slugify
+import homeassistant.util.color as color_utils
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_FEATURES_RGB = (
+    SUPPORT_BRIGHTNESS | SUPPORT_COLOR | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT
+)
+SUPPORT_FEATURES_DIM = SUPPORT_BRIGHTNESS
+SUPPORT_FEATURES_WHITE = SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {vol.Required(CONF_HOST): cv.string, vol.Required(CONF_NAME): cv.string}
+)
+
+# set poll interval to 15 sec because of changes from external to the bulb
+SCAN_INTERVAL = timedelta(seconds=15)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the WiZ Light platform from legacy config."""
+    # Assign configuration variables.
+    # The configuration check takes care they are present.
+    ip_address = config[CONF_HOST]
+    try:
+        bulb = wizlight(ip_address)
+        # Add devices
+        async_add_entities([WizBulb(bulb, config[CONF_NAME])], update_before_add=True)
+    except WizLightConnectionError:
+        _LOGGER.error("Can't add bulb with ip %s.", ip_address)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the WiZ Light platform from config_flow."""
+    # Assign configuration variables.
+    bulb = hass.data[DOMAIN][entry.unique_id]
+    wizbulb = WizBulb(bulb, entry.data.get(CONF_NAME))
+    # Add devices with defined name
+    async_add_entities([wizbulb], update_before_add=True)
+
+    # Register services
+    async def async_update(call=None):
+        """Trigger update."""
+        _LOGGER.debug(
+            "[wizlight %s] update requested", entry.data.get(CONF_HOST),
```

Comment:
I think this can all fit on a single line. This might fix the error with black.
_LOGGER.debug("[wizlight %s] update requested", entry.data.get(CONF_HOST))

Reply:
Changed .... Thank you. I missed that line 😉